fmpz.h

```diff
@@ -409,6 +409,8 @@ FLINT_DLL int fmpz_cmp_si(const fmpz_t f, slong g);
 
 FLINT_DLL int fmpz_cmpabs(const fmpz_t f, const fmpz_t g);
 
+FLINT_DLL int fmpz_cmp2abs(const fmpz_t f, const fmpz_t g);
```

Comment:
Seems awfully bespoke. Is this just a helper function that could go in some other file, or does it have general applicability for others?

Reply:
It is no more bespoke than fmpz_abs_lbound_ui_2exp. fmpz_smod (the old one) can use it instead of doing all of the stuff that it does.

Reply:
ok.